cmd/podman/main.go

```diff
@@ -151,6 +151,11 @@ func main() {
 			Usage: "used to pass an option to the storage driver",
 		},
 	}
+	if _, err := os.Stat("/etc/containers/registries.conf"); err != nil {
+		if os.IsNotExist(err) {
+			logrus.Warn("unable to find /etc/containers/registries.conf. some podman commands may be limited")
```

Comment:
Can you mention this specifically relates to images? "Some image related functions may be limited"?